src/Files.App/UserControls/StatusBarControl.xaml

```diff
@@ -137,9 +138,37 @@
 				Visibility="{x:Bind SelectedItemsPropertiesViewModel.ItemSizeVisibility, Mode=OneWay}" />
 		</StackPanel>
 
+		<!--  Open in VS Code Button  -->
+		<Button
+			x:Name="OpenInVSCodeButton"
+			Grid.Column="1"
+			Height="24"
+			Padding="8,0,8,0"
+			VerticalAlignment="Center"
+			x:Load="{x:Bind converters:MultiBooleanConverter.AndNotConvert(Commands.OpenInVSCode.IsExecutable, Commands.OpenInVS.IsExecutable), Mode=OneWay}"
+			Background="Transparent"
+			BorderBrush="Transparent"
+			Command="{x:Bind Commands.OpenInVSCode}"
+			Content="{x:Bind Commands.OpenInVSCode.Label}"
+			ToolTipService.ToolTip="{x:Bind Commands.OpenInVSCode.LabelWithHotKey, Mode=OneWay}" />
+
+		<!--  Open in VS Button  -->
+		<Button
+			x:Name="OpenInVSButton"
+			Grid.Column="1"
+			Height="24"
+			Padding="8,0,8,0"
+			VerticalAlignment="Center"
+			x:Load="{x:Bind Commands.OpenInVS.IsExecutable, Mode=OneWay}"
+			Background="Transparent"
+			BorderBrush="Transparent"
+			Command="{x:Bind Commands.OpenInVS}"
+			Content="{x:Bind Commands.OpenInVS.Label}"
+			ToolTipService.ToolTip="{x:Bind Commands.OpenInVS.LabelWithHotKey, Mode=OneWay}" />
```

Comment:
Shouldn't those be HyperlinkButtons?

Reply:
I'm not sure, they're not hyperlinks, they start a powershell command

Reply:
Buttons are fine